FNA.Core.csproj

```diff
@@ -10,6 +9,30 @@
 		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
 		<GenerateAssemblyInfo>false</GenerateAssemblyInfo>
 	</PropertyGroup>
+	<PropertyGroup>
+		<IsPackable>true</IsPackable>
+		<Version>23.07</Version>
+		<Description>Accuracy-focused XNA4 reimplementation for open platforms</Description>
+		<Copyright>Copyright (c) 2009-2023</Copyright>
+		<Authors>flibitijibibo</Authors>
+		<PackageProjectUrl>https://fna-xna.github.io/</PackageProjectUrl>
+		<PublishRepositoryUrl>true</PublishRepositoryUrl>
+		<PackageReadmeFile>README.md</PackageReadmeFile>
+		<PackageIcon>FNA.png</PackageIcon>
+		<PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
+	</PropertyGroup>
+	<PropertyGroup>
+		<IncludeSymbols>true</IncludeSymbols>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+		<EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
Another way to use the source is using the GitHub source link:
```csproj
<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
``` 
see [documentation](https://github.com/dotnet/sourcelink#githubcom-and-github-enterprise). This does a package reference to the project though, but it's how you can point to the GitHub code directly.

Reply:
I have been using the .NET 8 SDK, which [imports SourceLink automatically](https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8#source-link). You're right that the package is needed for earlier SDKs, and the GitHub Actions images probably won't have .NET 8 for a while, though.

All `EmbedUntrackedSources` does here is embed `AssemblyInfo.g.cs` (and probably some other equally meaningless files), I can probably remove it...

Reply:
That's pretty cool! I wasn't sure if .NET 8 SDK pulled the GitHub Source Link but that does seem to be the case. `EmbedUntrackedSources` is probably okay here.

> the GitHub Actions images probably won't have .NET 8 for a while

I _think_ we can just specify the .NET 8 SDK in the GitHub Actions (`8.0.x` should pull the RC1 version).